neuro_san/internals/graph/persistence/registry_manifest_restorer.py

```diff
@@ -66,13 +66,13 @@ def __init__(self, manifest_files: Union[str, List[str]] = None):
         self.logger = logging.getLogger(self.__class__.__name__)
 
     # pylint: disable=too-many-locals
-    def restore_from_files(self, file_references: Sequence[str]) -> Dict[str, AgentToolRegistry]:
+    def restore_from_files(self, file_references: Sequence[str]) -> Dict[str, AgentNetwork]:
```

Comment:
* What we restore from the RegistryManifestRestorer is now AgentNetworks instead of AgentToolRegistry-s